message_definitions/v1.0/ras_a.xml

```diff
@@ -16,6 +16,67 @@
+      </entry>
+      <entry value="37" name="MAV_CMD_DO_FOLLOW_POI" hasLocation="false" isDestination="false">
+        <description>Begin following a target specified by POI ID.  A POI Unique ID (UID) is a 64-bit field, and so to fully represent that UID, the big endian 64-bit POI UID is split into 2 words; param1: upper 32-bit word, param2: lower 32-bit word.</description>
+        <param index="1" label="POI UID Upper 32 Bits">The upper 32 bits of the UID from a POI_REPORT.</param>
+        <param index="2" label="POI UID Lower 32 Bits">The lower 32 bits of the UID from a POI_REPORT</param>
+        <param index="3" label="Mode Flags" enum="FOLLOW_POI_MODE_FLAGS">Bitmask of option flags.</param>
+        <param index="4" label="Altitude" units="mm or degE7">Altitude field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="5" label="Azimuth" units="degE7">Azimuth field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="6" label="Range" units="mm">Range field, behavior to be interpreted according to Mode Flags.</param>
+        <param index="7" label="Time to Return" units="s" minValue="0">Time to return in which the Autopilot should enter return to comms behavior. If 0, vehicle will immediaately enter return to comms behavior after losing comms with GCS</param>
+      </entry>
+      <entry value="38" name="MAV_CMD_DO_FOLLOW_NONE" hasLocation="false" isDestination="false">
+        <description>Stop following a target</description>
+        <param index="1" label="Return to comms" minValue="0" maxValue="1" increment="1">If in loss of comms state, 0: Begin return to comms behavior, 1: Hold Position.</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4">Reserved</param>
+        <param index="5">Reserved</param>
+        <param index="6">Reserved</param>
+        <param index="7">Reserved</param>
+      </entry>
+    </enum>
+    
+    <enum name="FOLLOW_POI_MODE_FLAGS" bitmask="true">
+      <entry value="1" name="ALTITUDE_MODE_AMSL">
+        <description>Altitude to be interpreted as absolute AMSL</description>
+      </entry>
+      <entry value="2" name="ALTITUDE_MODE_RELATIVE">
+        <description>Altitude to be interpreted as relative to follow target</description>
+      </entry>
+      <entry value="4" name="ALTITUDE_MODE_RELATIVE_ELEVATION_ANGLE">
+        <description>Altitude interpreted as an elevation angle, ie the angle from POV of target between horizon and vehicle</description>
+      </entry>
+      <!-- Fields 4-8 reserved for future altiude modes-->
+      <entry value="256" name="AZIMUTH_MODE_FIXED_YAW">
+        <description>Azimuth to be interpreted as the vehicles fixed yaw in local NED frame. Autopilot will update the relative angle to subject to maintain this fixed yaw heading.</description>
```

Comment:
should we consider the magnetic north / global frame instead?

Reply:
I have no opinion on this, provided there is an easy way to map between the two systems.